### DISCUSSION TITLE
Allow custom file system mount points when installing quota

### DISCUSSION THREAD

**Discussion by ghost** [swizzin/swizzin#358] [Feature Requests]
Don't limit user to /home or /
Allow user to enter custom mount points.

**Comment by flying-sausages** [comment]
I’m 99% sure about this, so I might have some tidbits wrong, but... The options displayed aren’t paths but mount points. Quota works on a scope of an entire file system, not a directory.

**Comment by ghost** [comment]
Sorry I meant allow custom mount points.
